frontend/views/containers/proposals/DistributionDate.vue

```diff
@@ -0,0 +1,187 @@
+<template lang='pug'>
+  proposal-template(
+    ref='proposal'
+    :title='L("Change distribution date")'
+    :disabled='$v.form.$invalid'
+    :maxSteps='config.steps.length'
+    :currentStep.sync='ephemeral.currentStep'
+    variant='changeDistributionDate'
+    :skipToPropose='skipToProposeDistributionDate'
+    @submit='submit'
+  )
+
+    label.field(v-if='ephemeral.currentStep === 0' key='0')
+      i18n.label New distribution date
+      .inputgroup.selectbox(
+        :class='{ error: $v.form.distributionDate.$error }'
+        v-error:distributionDate=''
+      )
+        select.select(
+          ref='distributionDate'
+          :aria-label='L("Choose your group\'s distribution date")'
+          name='distributionDate'
+          required
+          v-model='$v.form.distributionDate.$model'
+        )
+          i18n(tag='option' disabled value='') Choose your group's distribution date
+          option(
+            v-for='(item, index) in ephemeral.distributionDayRange'
+            :key='index'
+            :value='item'
+          ) {{ humanDate(item, { month: 'long', year: 'numeric', day: 'numeric' }) }}
+      i18n.helper(:args='{currentDistributionDate}') Current distribution date is on {currentDistributionDate}.
+
+    template(#skipToProposeFooter='' v-if='skipToProposeDistributionDate')
```

Comment:
It seems you haven't noticed that `ProposalTemplate.vue` already has a slot for footer in there. So you can use `template(#footer='')` instead here. (and `skipToProposeFooter` here and there should be removed)

Reply:
I saw the `template(#footer='')`. But I didn't want to repeat the whole content of that footer. I just wanted to add additional footer only when it's needed.

Reply:
@Silver-IT 
What I meant was,

in `DistributionDate.vue`

```
template(#footer='' v-if='skipToProposeDistributionDate')
  .c-footer
    i.icon-vote-yea
    i18n(
      :args='LTags("strong")'
    ) The first distribution period is not started yet, so {strong_}this change will be immediate{_strong} (no voting required).
```

also in the relevant part of `ProposalTemplate.vue`
```
template(slot='footer' v-if='!isConfirmation')
  .c-footer
    i.icon-vote-yea
    span(v-if='shouldPropose' v-safe-html='footerVotingExplanation')
    i18n(
      v-else
      :args='LTags("strong")'
    ) Your group has less than 3 members, so {strong_}this change will be immediate{_strong} (no voting required).
```

This way, we can remove the `slot` in another `slot` pattern in `ProposalTemplate.vue`. The decision is up to you  though!

Reply:
I don't think we can use `footer` slot in `DistributionDate.vue` because it is from the `ModalTemplate`, not `ProposalTemplate.vue`.

Reply:
@SebinSong, in my honest point of view, it's better to leave it same and make no other changes more.
@taoeffect, what do you think?

Reply:
@Silver-IT I tried this change and it did work but if you don't agree, that's ok. up to your and Greg's opinion